README.md

```diff
@@ -162,6 +162,26 @@ const { items, pagination } = StoresCollection.getPaginated({
 });
 ```
 
+### Options
```

Comment:
Does this work for Meteor.users collection? I don't think so, right? It should be warned here.